### PR TITLE
Created a german translation

### PR DESCRIPTION
**Pull request by Kllngii**

### CODE REVIEW

File: src/renderer/i18n/de.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "rememberMe": "Logindaten merken",
+    "thisisimportant": "Das ist wichtig!",
+    "login/letsgetstarted": "Alles bereit! - Lass uns anfangen!",
+    "login/entercredentials": "Bitte fülle die Felder mit deinen Ubisoft-Logindaten aus.",
+    "login/whydoyouneed": "Warum benötigen wir die Logindaten?",
+    "login/popup/paragraph1": "Um diese App zu nutzen brauchen wir deine <b> Ubisoft-Logindaten</b>.",
+    "login/popup/paragraph2": "Ohne Anmeldung können wir keine Daten von der Ubisoft-API bekommen. Wir werden diese Daten <em>allein</em> für diesen Zweck nutzen.<br />",
+    "login/popup/paragraph2link": "Sie können selbst sehen, was wir mit den Daten machen {codeBase}, dies ist ein Open-Source Projekt.",
+    "login/popup/paragraph3": "Wenn sie nicht ihre Accoutdaten eintragen wollen, können sie auch einen neuen {ubiAccount} erstellen und diese Daten hier eintragen.",
+    "login/popup/paragraph4": "Egal, welchen Weg sie wählen wir empfehlen <em>stark</em>, dass sie <b>2-Faktor-Authentifizierung</b> für alle ihre Accounts verwenden und ein <em>starkese/em <b>Passwort</b> wählen."
```
Comment (LaxisB) [inline src/renderer/i18n/de.json]: small typo here 
-> starkes
-> closing </em> is borked

**Comment by LaxisB** [comment]
can you formulate the paragraphs with `du`, instead of `sie`? the formal talk doesn't fit us well :)

**Comment by MarcoPixel** [comment]
@tiehm or maybe "Angemeldet bleiben", both sounds a little bit better in my opinion.

**Comment by Kllngii** [comment]
@tiehm @MarcoPixel I think "Angemeldet bleiben" is the better one.

**Comment by LaxisB** [comment]
Läuft

**Pull request merged**